changelog.md

```diff
@@ -10,6 +10,7 @@
 
 #### Bug fixes
 
+* Fixed bug in `Kymo` which resulted in erroneous timestamps and line times after slicing the Kymograph by time. The reason for this was imprecision in the timestamp reconstruction that occurred when slicing the data. This in turn led to an erroneous reconstruction of the timestamps. **As a result, any downstream analysis that relies on the time axis of a Kymograph post-slicing cannot be trusted (MSD analysis, plotting, tracking, etc) in pylake v0.8.2 and v0.9.0.** Only those two versions are affected. Note that regular image reconstruction was not affected.
```

Comment:
We might add at the end something like:
"Timestamps of un-sliced `Kymo`s constructed directly from the .h5 files are also unaffected"
just to really nail down the point that this bug is only when you slice